cEP-0016.md

```diff
@@ -0,0 +1,180 @@
+# Git Commit Content Inspection
+
+| Metadata |                                                   |
+| -------- | ------------------------------------------------- |
+| cEP      | 16                                                |
+| Version  | 1.0                                               |
+| Title    | Git commit content inspection                     |
+| Authors  | Kriti Rohillla <mailto:kritirohilla567@gmail.com> |
+| Status   | Implementation due                                |
+| Type     | Process                                           |
+
+## Abstract
+
+This document describes coala process rules which will be enforced by the
+GitCommitBear. Some new changes will be introduced in the implementation of
+GitCommitBear based on which the commits made to coala will be detected and
+verified.
+
+## Introduction
+
+The GitCommitBear at present performs a check on the content of regular commits
```

Comment:
Please do a quick review of all .coafiles in active repositories on GitHub & GitLab for current use of GitCommitBear and document this as "Background" in your cEP.

Note that coala can not rely on Travis for CI, because of the serious bug in GitCommitBear unsolved for years.  And thus we are reliant on CircleCI.

Reply:
Not done yet.

Reply:
Also you have removed all mention of Travis, when the most important issue for this project is a Travis specific bug.